pages/[user-slug]/[event-type-slug].tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import AccessTimeTwoToneIcon from '@mui/icons-material/AccessTimeTwoTone';
+import PublicTwoToneIcon from '@mui/icons-material/PublicTwoTone';
+import KeyboardArrowDownOutlinedIcon from '@mui/icons-material/KeyboardArrowDownOutlined';
+
+import DatePicker from '../../components/shared/DatePicker';
```

Comment:
can you create an alias for that in the tsconfig? that we can access it via `@components/shared/DatePicker`